presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/Hadoop.java

```diff
@@ -59,9 +59,19 @@ public void extendEnvironment(Environment.Builder builder)
     {
         builder.addContainer("hadoop-master", createHadoopMaster());
 
-        builder.configureContainer("presto-master", container -> container
-                .withFileSystemBind(dockerFiles.getDockerFilesHostPath("common/hadoop/hive.properties"), CONTAINER_PRESTO_HIVE_PROPERTIES, READ_ONLY)
-                .withFileSystemBind(dockerFiles.getDockerFilesHostPath("common/hadoop/iceberg.properties"), CONTAINER_PRESTO_ICEBERG_PROPERTIES, READ_ONLY));
+        builder.configureContainer("presto-master", container -> {
+            container
+                    .withFileSystemBind(dockerFiles.getDockerFilesHostPath("common/hadoop/hive.properties"), CONTAINER_PRESTO_HIVE_PROPERTIES, READ_ONLY)
+                    .withFileSystemBind(dockerFiles.getDockerFilesHostPath("common/hadoop/iceberg.properties"), CONTAINER_PRESTO_ICEBERG_PROPERTIES, READ_ONLY);
+
+            if (System.getenv("HADOOP_PRESTO_INIT_SCRIPT") != null) {
```

Comment:
nit: extract to variable

Reply:
i thought this is concise enough that there is no need